docusaurus/docs/cloud/advanced/email.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Email Provider
+displayed_sidebar: cloudSidebar
+description: Configure Strapi Cloud to use a third-party email provider.
+canonicalUrl: https://docs.strapi.io/cloud/advanced/email.html
+---
+
+# Email Provider
+
+Strapi Cloud comes with a basic email provider out of the box. However, you can also configure it to utilize a third-party email provider, if needed.
+
+:::prerequisites
+
+- A local Strapi project running on `v4.8.2+`.
+- Credentials for a third-party email provider (see [Strapi Market](https://market.strapi.io/)).
+
+:::
+
+## Configuration
+To configure a third-party email provider, in your Strapi project, create a `./config/env/production/plugins.js` or `./config/env/production/plugins.ts` file with the following content:
+
+```js
```

Comment:
We should provide example files for both JS and TS here as they will be slightly different